### PR TITLE
Updating static pages for updated details

### PR DESCRIPTION
**Pull request by SpartanApple**
Updated pages and fixed dead links.

### CODE REVIEW

**Comment by modernNeo** [comment]
is this for a school assignment or something?

File: csss-site/src/csss/templates/csss/header.html [hunk 1]
```diff
@@ -137,12 +137,12 @@
...
-							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href="http://esss.ca/" >ESSS</a>
-							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href="https://msess.ca/index.html" >Mechs</a>
-							<a target="_blank"  class="navbar-item csss_header_more_sub_items_indent" href='https://www.facebook.com/ssss.sfu/' >SSSS</a>
-							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href='http://www.sfu.ca/clubs/wie.html' >WiE</a>
+							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href="https://www.sfu.ca/computing/wics/">Women in Computing Science</a>
+							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href="https://sfumsu.github.io/" >Mathematics Student Union</a>
+							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href="http://esss.ca/" >Engineering Science Student Society</a>
+							<a target="_blank" class="navbar-item csss_header_more_sub_items_indent" href="https://msess.ca/index.html" >Mechatronics Systems Engineering Student Society</a>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/header.html:143]: it's apparently https://www.msess.ca/

File: csss-site/src/static_pages/templates/static_pages/bursaries.html [hunk 2]
```diff
@@ -7,48 +7,94 @@
...
+<div><a href="https://awards-search.sfu.ca/Home/AwardDetail/61829">Computing Science Undergraduate Student Society Award</a></div>
+
+<a href="https://sfucsss.org">Computing Science Student Society Website</a>
+
+
+<h2>Bit Quill Technologies Scholarship for Women in Computing Science</h2>
+		
+<p>Two or more scholarships, valued at $2,500 each, will be granted annually in any term. The scholarships will be awarded to undergraduate female student majoring in Computing Science on the basis of full-time enrollment and excellent academic standing. </p>
+
+<div><a  ref="https://awards-search.sfu.ca/Home/AwardDetail/62186">Bit Quill Technologies Scholarship for Women in Computing Science</a></div>
```
Comment (modernNeo) [inline csss-site/src/static_pages/templates/static_pages/bursaries.html:29]: forgot the `h` in `hre`

File: csss-site/src/static_pages/templates/static_pages/bursaries.html [hunk 3]
```diff
@@ -7,48 +7,94 @@
...
+
+<a href="https://awards-search.sfu.ca/Home/AwardDetail/62489">Peng Tu Ge Undergraduate Scholarship for Women in Engineering Science or in Computing Science</a>
+
+<h2>Matthew LeDuc Memorial Scholarship in Computing Science</h2>
+
+ Awarded annually on the basis of academic achievement to a Computing Science major, with demonstrated excellence in the field of computer graphics. The award will be made by the Senate Undergraduate Awards Adjudication Committee upon the nomination of the Director, School of Computing Science. 
+
+<a href="https://awards-search.sfu.ca/Home/AwardDetail/62128">Matthew LeDuc Memorial Scholarship in Computing Science</a>
```
Comment (modernNeo) [inline csss-site/src/static_pages/templates/static_pages/bursaries.html:77]: this isn't on a newline like the above links

File: csss-site/src/static_pages/templates/static_pages/bursaries.html [hunk 4]
```diff
@@ -7,48 +7,94 @@
...
+
+<h2>Mark and Nancy Brooks Computing Science Innovation Award</h2>
+
+ Granted to a computing science student in good academic standing who demonstrates exceptional accomplishment, promise or innovation in the area of computing science outside classroom work. The application should include a description of the interest or innovative ideas that student is considering. The award will be made by the Senate Undergraduate Awards Adjudication Committee on the nomination of the Director, School of Computing Science. 
+
+<a href="https://awards-search.sfu.ca/Home/AwardDetail/62094">Mark and Nancy Brooks Computing Science Innovation Award</a>
```
Comment (modernNeo) [inline csss-site/src/static_pages/templates/static_pages/bursaries.html:83]: this isn't on a newline like the above links

**Review by modernNeo — CHANGES_REQUESTED** [review]
some bugs to fix

**Comment by SpartanApple** [comment]
> is this for a school assignment or something?

No, just keeping the information up to date.

File: csss-site/src/csss/templates/csss/header.html [hunk 5]
```diff
@@ -91,14 +91,13 @@
 						{% endif %}
 						<div class="navbar-dropdown">
 							<a href="{{URL_ROOT}}events/regular_events" class="navbar-item">Regular Events</a>
+							<a href="{{URL_ROOT}}events/tech_fair/2022" class="navbar-item">Tech Fair 2022</a>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/header.html:94]: this does not exist?

File: csss-site/src/csss/templates/csss/header.html [hunk 6]
```diff
@@ -91,14 +91,13 @@
...
 							<a href='{{ URL_ROOT}}events/frosh/' class="navbar-item">CSSS FroshWeek</a>
 							<a href="{{URL_ROOT}}events/mm/2022" class="navbar-item">Mountain Madness 2022</a>
+							<a href="{{URL_ROOT}}events/mm/2022" class="navbar-item">Mountain Madness 2022</a>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/header.html:97]: isn't this a duplicate of the above line?

File: csss-site/src/static_pages/templates/static_pages/bursaries.html [hunk 7]
```diff
@@ -7,48 +7,94 @@
...
+
+<h2>Maliha & Amina Bai Ropani Memorial Scholarship for Women in Computing Science</h2>
+At least one scholarship, valued at a minimum of $1,000, will be granted annually in any term to a female undergraduate student who meets the following criteria:-is enrolled in a full-time Bachelor's degree program in the School of Computing Science;-is majoring in Computing Science; and,-is in excellent academic standing.The scholarship will be administered by the office of Financial Aid and Awards and granted by the Senate Undergraduate Awards Adjudication Committee. 
+
+<a href="https://awards-search.sfu.ca/Home/AwardDetail/62483">Maliha & Amina Bai Ropani Memorial Scholarship for Women in Computing Science</a>
```
Comment (modernNeo) [inline csss-site/src/static_pages/templates/static_pages/bursaries.html:88]: this isn't on a newline like the above links

File: csss-site/src/static_pages/templates/static_pages/bursaries.html [hunk 8]
```diff
@@ -7,48 +7,94 @@
...
+
+
+<h2>(External Award) Irving K. Barber BC Scholarship Society Women in Technology Scholarships</h2>
+
+You must have completed a minimum of one-year of post-secondary studies (30 credits) leading to an undergraduate degree, diploma or certificate in Computer/Data Science, Engineering, or Mathematics, and be continuing in your studies at a BC public post-secondary institution for at least one additional year.
 
+Women in Technology Scholarships and Indigenous Women in Technology Scholarships are awarded on the basis of academic achievement (Minimum B+ GPA required), and a demonstrated commitment to the pursuit of a career in the Technology Sector.
 
+<a href="https://www.ikbbc.ca/women-in-tech/about-this-scholarship/">Irving K. Barber BC Scholarship Society Women in Technology Scholarships</a>
```
Comment (modernNeo) [inline csss-site/src/static_pages/templates/static_pages/bursaries.html:97]: this isn't on a newline like the above links

File: csss-site/src/static_pages/templates/static_pages/bursaries.html [hunk 9]
```diff
@@ -7,48 +7,94 @@
             <div class="content" >
                 <h1>Scholarships, Awards, and Bursaries</h1>
```
Comment (modernNeo) [inline csss-site/src/static_pages/templates/static_pages/bursaries.html:8]: please make all external links on this page open on a new tab

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/csss/templates/csss/header.html (hunk 5 above)
Comment (SpartanApple) [reply csss-site/src/csss/templates/csss/header.html:94]: Oops... part of my next PR. Should be out today.

**Pull request merged**